src/main/webapp/app/modeling/manage/modeling-exercise-timeline/modeling-exercise-timeline.component.html

```
@@ -0,0 +1 @@
+<jhi-exercise-timeline [timelineItems]="timelineItems" (timelineStatusChange)="this.timelineStatus.emit($event)" />
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)